models/fixtures/webauthn_credential.yml

```diff
@@ -0,0 +1,8 @@
+- id: 1
+  name: "WebAuthn credential"
+  user_id: 1
```

Comment:
In u2f_registration.yml the user_id was 32. Changing this to 1 is causing the CI to fail.